.github/workflows/test-all-packages.yml

```diff
@@ -221,6 +221,7 @@ jobs:
         with:
           datadog-token: ${{ secrets.DATADOG_API_KEY }}
           codecov-token: ${{ secrets.CODECOV_TOKEN }}
+          gcp-credentials: ${{ secrets.GCP_CREDENTIALS }}
```

Comment:
i think its better this way since:
a) it follows the conventions of other tokens
b) it allows us to explicitly decide to pass the token or not. In case we don't pass it, the code that uses it does not execute